Polygon_mesh_processing/test/Polygon_mesh_processing/orient_polygon_soup_test.cpp

```diff
@@ -134,9 +136,70 @@ int test_orient(const bool save_oriented) {
   return 0;
 }
 
+
+template <class K, class Tag>
+int test_pipeline()
+{
+  typedef typename K::Point_3 Point_3;
+  typedef CGAL::Polyhedron_3<K> Polyhedron;
+
+  std::vector<Point_3> points;
+  std::vector< std::vector<std::size_t> > polygons;
+  Polyhedron ref1;
+
+  shuffle_off("data/elephant.off", "elephant-shuffled.off");
+  std::ifstream input("elephant-shuffled.off");
+  if ( !input || !read_soup<K>(input, points, polygons)){
+    std::cerr << "Error: can not shuffled file.\n";
+    return 1;
+  }
+  input.close();
+  input.open("data/elephant.off");
+  if ( !input || !(input >> ref1)){
+    std::cerr << "Error: can not read reference file.\n";
+    return 1;
+  }
+  input.close();
+  CGAL::Polygon_mesh_processing::orient_triangle_soup_with_reference_triangle_mesh<Tag>(ref1, points, polygons);
+
+  CGAL::Polygon_mesh_processing::duplicate_incompatible_edges_in_polygon_soup(points, polygons);
+
+  Polyhedron poly;
+  CGAL::Polygon_mesh_processing::polygon_soup_to_polygon_mesh(
+        points, polygons, poly);
+  typedef typename boost::property_map<Polyhedron, CGAL::dynamic_face_property_t<std::size_t> >::type Fccmap;
```

Comment:
remove extra `typename`

Reply:
`Polyhedron` is a templated type, so I'm quite sure this `typename` is not extra.

Reply:
it's not...
see above : ` typedef CGAL::Polyhedron_3<K> Polyhedron;`

Reply:
I tried without just to be sure, and it won't compile.

Reply:
ok thank you